prac2-pages/Trisha/login_page.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/cupertino.dart';

```

Comment:
There is no need to import cupertino.dart, as it is already included in the material package. 
Kindly remove it, as unused imports increase the size of the file.